.github/workflows/pr-request.yaml

```diff
@@ -20,6 +20,20 @@ on:
       - 'main'
 
 jobs:
+  lint:
+    name: Lint Markdown with Vale
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v4
+
+      - name: Run Vale linting
+        uses: errata-ai/vale-action@d027270961a47e2261440e037ef59f1ecafb1681 #v2.1.1
```

Comment:
Can you verify the hash? 
`https://github.com/errata-ai/vale-action/commits/v2.1.1` is pointing `d89dee975228ae261d22c15adcd03578634d429c`.

I couldn't find `d027270961a47e2261440e037ef59f1ecafb1681` on the repo.